tools/clang/lib/SPIRV/SpirvEmitter.cpp

```diff
@@ -977,6 +977,10 @@ void SpirvEmitter::doFunctionDecl(const FunctionDecl *decl) {
   // Create all parameters.
   for (uint32_t i = 0; i < decl->getNumParams(); ++i) {
     const ParmVarDecl *paramDecl = decl->getParamDecl(i);
+    if (spvContext.isHS() && decl == patchConstFunc &&
```

Comment:
Please add a comment about why this parameter should be skipped. e.g.
```cpp
// Since the output patch used in hull shaders is translated to a variable with Workgroup storage
// class, there is no need to pass the variable as function parameter in SPIR-V.
```

Reply:
done